packages/components/src/select-control/stories/index.tsx

```diff
@@ -0,0 +1,88 @@
+/**
+ * External dependencies
+ */
+import type { ComponentMeta, ComponentStory } from '@storybook/react';
+import type { ComponentProps } from 'react';
+
+/**
+ * WordPress dependencies
+ */
+import { useState } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import SelectControl from '..';
+
+const meta: ComponentMeta< typeof SelectControl > = {
+	title: 'Components/SelectControl',
+	component: SelectControl,
+	argTypes: {
+		help: { control: { type: 'text' } },
+		label: { control: { type: 'text' } },
+		value: { control: { type: null } },
```

Comment:
Currently, the `suffix` property has a "Set Object" control that breaks the story when clicked. We could either:

- disable it entirely (e.g. set the type to `null`)
- change it to a `string`
- change it to a `select` and prepare a few options (e.g. a react element, a string, or nothing)

WDYT ?

Reply:
Nice catch. I'll just go with a quick `string` for this one (e00184f). I couldn't really think of a single realistic or advisable use case for prefix/suffix in this component, so I don't feel comfortable giving explicit suggestions 😂